src/server/server_family_test.cc

```diff
@@ -275,7 +275,9 @@ TEST_F(ServerFamilyTest, ClientTrackingMulti) {
 
   Run({"MULTI"});
   auto resp = Run({"CLIENT", "TRACKING", "ON"});
-  EXPECT_THAT(resp, ArgType(RespExpr::STRING));
+  EXPECT_THAT(resp.GetString(), "QUEUED");
+  resp = Run({"CLIENT", "KILL", "127.0.0.1:6380"});
```

Comment:
please add a comment saying it's needed by sentinel so that we will know the context

Reply:
I did that but I also added tests for `all client` commands since all the client commands we implement are allowed in `MULTI/EXEC` blocks and I think we should have a compatibility test for all of them (I did introduce that see last commit)